### ISSUE TITLE
Fatal error: Cannot use 'true' as class name as it is reserved

### ISSUE THREAD

**Issue by aj-norman** [mockery/mockery#1269]
Hi,

I'm getting an issue similar to #1149 when using the `true` return type in PHP 8.2. the error produced the following in terminal:

```
PHP Fatal error:  Cannot use 'true' as class name as it is reserved in /private/var/folders/s6/mt33vjsn36q1jkdprflfh5cr0000gn/T/MockeryKoER9D on line 1
```

## Env details:
 - PHP 8.2
 - Laravel 10.x
 - PHPUnit 10
 - Mockery 1.6.1

## Steps to reproduce: 

I have a method similar to the following:

```php
<?php

class Example
{
    public function testMethod(): true
    {
        // ...
    }
}
```

And in test I am doing the following:

```php
<?php
// ...

$mock = Mockery::mock(Example::class);
$mock->shouldHaveReceived("testMethod")->once();

// ...
```

Removing the `true` return type allows the test to pass with the same test code

**Comment by ghostwriter** [comment]
Hey @aj-norman,

This issue has been resolved via #1271 

Thank you for your contribution.

**Comment by aj-norman** [comment]
Thanks for fixing this
